Sources/Sentry/SentryCrashReportConverter.m

```diff
@@ -307,15 +307,13 @@ - (SentryFrame *)stackFrameAtIndex:(NSInteger)frameIndex inThreadIndex:(NSIntege
         = (uintptr_t)[frameDictionary[@"instruction_addr"] unsignedLongLongValue];
     NSDictionary *binaryImage = [self binaryImageForAddress:instructionAddress];
     SentryFrame *frame = [[SentryFrame alloc] init];
-    frame.symbolAddress = sentry_formatHexAddress(frameDictionary[@"symbol_addr"]);
+    // The symbol address is not known, so it defaults to 0
+    frame.symbolAddress = @"0x0000000000000000";
```

Comment:
`m`: We usually then set these things to `nil` and let the backend set default values, because it gives us more flexibility. Or do we absolutely need to set this to `@"0x0000000000000000"`?